src/services/ramp-hooks.js

```diff
@@ -386,12 +388,12 @@ export const useVideoJSPlayer = ({
 
       // Update player status in state only when pause is initiate by the user
       player.controlBar.getChild('PlayToggle').on('pointerdown', () => {
-        handlePause();
+        handlePause(player.paused());
```

Comment:
I'm a bit confused.  Should this be `player.paused()` or `!player.paused()`?  The `handlePause` function calls this argument `isPlaying` which seems like it would be the opposite of `player.paused()`.

Reply:
Yes, this feels wrong at a first glance 🙂 Because we expect the player status to be changed immediately upon the user interaction, but it seems these event handlers capture the player status before it gets altered with the respective action. And for that reason this works as expected by just using `player.paused()`.

For an example;
1. Using `PlayToggle` button:
  - Initially `player.paused()` = `true`
  - Click `PlayToggle` button => `player.paused()` = `true` from initial value before this action alters the player's status; `isPlaying` = `true`
  - Click `PlayToggle` button again => `player.paused()` = `false`; `isPlaying` = `false`

2. Using pointer on top of video:
  - Initially `player.paused()` = `true`
  - Click on top of video player => this event is captured by `BigPlayButton` overlay, therefore not captured in these event handlers. But this is captured in the `play` event handler in `VideoJSPlayer` module and it sets `isPlaying` = `true`
  - Click on top of video player again => `player.paused()` = `false`; `isPlaying` = `false`

I tested this again with logs, to make sure it works as expected for both scenarios where there are inaccessible items in the Manifest and all playable items in the Manifest.